scripts/data_ingestion/download_external_data.py

```
@@ -0,0 +1,104 @@
+    
+    if args.all or args.lahman:
+        results["lahman"] = download_lahman()
+    
+    if args.all or args.fangraphs:
+        results["fangraphs"] = download_fangraphs()
+    
+    if args.all or args.statcast:
+        results["statcast"] = download_statcast()
+    
+    if args.all or args.baseball_savant:
+        results["baseball_savant"] = download_baseball_savant()
```

Ruff (0.15.10)

[error] Undefined name `download_baseball_savant`

(F821)